hypershift-operator/controllers/nodepool/nodepool_controller.go

```diff
@@ -182,13 +182,15 @@ func (r *NodePoolReconciler) Reconcile(ctx context.Context, req ctrl.Request) (c
 		return result, err
 	}
 
+	setStatusCondition(&nodePool.Status.Conditions, generateReconciliationActiveCondition(nodePool.Spec.PausedUntil, nodePool.Generation))
```

Comment:
is there any reason why this is isolated here instead of along with where we set all the other condiions?

Reply:
Good point - this is just because I refactored things based on your earlier feedback - I'll move it, thanks!